packages/lesswrong/components/review/LatestReview.tsx

```diff
@@ -47,22 +48,24 @@ const LatestReview = ({classes}) => {
   const comment = commentResults[0]
   if (!comment.post) return null
 
-  return <ErrorBoundary><div className={classes.root} {...eventHandlers} >
-    <LWPopper
-      open={hover}
-      anchorEl={anchorEl}
-      placement="bottom-start"
-      modifiers={{
-        flip: {
-          behavior: ["bottom-start"],
-          boundariesElement: 'viewport'
-        }
-      }}
-    >
-      <span className={classes.preview}>{<PostsPreviewTooltipSingleWithComment postId={comment.postId} commentId={comment._id}/>}</span>
-    </LWPopper>
-    <Link to={commentGetPageUrlFromIds({postId: comment.postId, commentId: comment._id, postSlug: comment.post.slug})} className={classes.lastReview}>Latest Review: <span className={classes.title}>{comment.post.title}</span></Link>
-  </div></ErrorBoundary>
+  return <ErrorBoundary><AnalyticsContext pageSubsectionContext="LatestReview">
```

Comment:
This should probably be set to track hover-events. @darkruby501 how do I do that?

Reply:
I believe there's tracking built into `withHover`, so should happen automatically, and AnalyticsContext will enable you to pick out those events.